tests/components/androidtv/test_media_player.py

```diff
@@ -968,3 +970,34 @@ async def test_androidtv_volume_set(hass):
         )
 
         patch_set_volume_level.assert_called_with(0.5)
+
+
+async def test_get_image(hass, hass_ws_client):
+    """Test taking a screen capture.
+
+    This is based on `test_get_image` in tests/components/media_player/test_init.py.
+    """
+    patch_key, entity_id = _setup(CONFIG_ANDROIDTV_ADB_SERVER)
+
+    with patchers.PATCH_ADB_DEVICE_TCP, patchers.patch_connect(True)[
+        patch_key
+    ], patchers.patch_shell("")[patch_key]:
+        assert await async_setup_component(hass, DOMAIN, CONFIG_ANDROIDTV_ADB_SERVER)
+
+    with patchers.patch_shell("11")[patch_key]:
+        await hass.helpers.entity_component.async_update_entity(entity_id)
+
+    client = await hass_ws_client(hass)
+
+    with patch("androidtv.basetv.BaseTV.adb_screencap", return_value=b"image"):
+        await client.send_json(
+            {"id": 5, "type": "media_player_thumbnail", "entity_id": entity_id}
+        )
+
+        msg = await client.receive_json()
+
+    assert msg["id"] == 5
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5357848)